parser.y

```diff
@@ -267,7 +265,7 @@ import (
 	zerofill		"ZEROFILL"
 	natural			"NATURAL"
 
-	/* The following tokens belong to UnReservedKeyword. */
+	/* The following tokens belong to UnReservedKeyword. Notice: make sure these tokens are contained in UnReservedKeyword. */
```

Comment:
🤔 So turns out this is already checked in `consistent_test.go`

Reply:
🤔but it don't take effect when 'node_id' not in keyword list, I don't know why

Reply:
Only put `node_id` in TiDBKeyword part (or other special parts) can take effect.